groups/sig-etcd/groups.yaml

```diff
@@ -23,10 +23,21 @@ groups:
       - jablair@redhat.com
       - siarkowicz@google.com
       - wenjiazhang@google.com
+      - ivan@vald.es
```

Comment:
@BenTheElder, one question. I noticed this is not in alphabetical order. Is this not required? Or should the linter have caught this?

Reply:
we tend to sort things to minimize churn but it shouldn't matter and these probably wont' be edited terribly often